Tools/Create-Azure-Sentinel-Solution/V2/createSolutionV2.ps1

```diff
@@ -97,10 +97,16 @@ function getParserDetails($solutionName)
             $parserTemplate = $templateContent.resources | Where-Object { $_.type -eq $parserResourceType.templateSpecParserType -or $_.type -eq $parserResourceType.workspaceType }
 
             if ($parserTemplate) {
-                if($parserTemplate.resources)
+                if($parserTemplate.PSobject.Properties.Name -contains "resources")
                 {
                     $parserTemplate = $parserTemplate.resources | Where-Object {$_.properties.category -eq "Samples" -and $_.type -eq $parserResourceType.normalParserType }
```

Comment:
Plz check the parser resource type condition logic it should be savedSearches or Microsoft.OperationalInsights/workspaces/savedSearches